tests/fixtures/typescript/basics/class-with-accessibility-modifiers.src.ts

```diff
@@ -0,0 +1,9 @@
+class Foo {
+  private bar : string;
+  public getBar () {
+    return this.bar;
+  }
+  protected setBar (bar : string) {
+    this.bar = bar;
+  }
+}
```

Comment:
Please add at least one example of methods and properties which have multiple TS modifiers on them:

e.g. 

```ts
class X {
  public static yo: string = 'Init';
  protected get bye() {
    return 'bye';
  }
}
```